tensorflow/lite/micro/tools/project_generation/create_tflm_tree.py

```diff
@@ -233,8 +257,10 @@ def main():
   # no longer be needed once all the downloads are switched over to bash
   # scripts.
   params_list = [
-      "make", "-f", "tensorflow/lite/micro/tools/make/Makefile",
-      "third_party_downloads"
+      "make", "-f",
```

Comment:
per our discssion, we will try and use `TENSORFLOW_ROOT` and `EXTERNAL_DIR` from `ar.makefile_options`.

Reply:
Done! Thanks for the suggestions.